### ISSUE TITLE
when completely swipe and close app interval stops 

### ISSUE THREAD

**Issue by Xaus** [ocetnik/react-native-background-timer#84]
when completely swipe and close app interval stops and i have to re run the app and the event (like click on a button) to start the interval

**Comment by getabatu** [comment]
+1

**Comment by xerotolerant** [comment]
I'm not the author, but this is how I would expect this to work. Swipe to close is OS level, the app isn't supposed to function at all once it is closed like that.

**Comment by TriptiPant01** [comment]
But background task means running in the background even the app is stopped isn't it?

**Comment by xerotolerant** [comment]
Not really, see [react-native docs](https://facebook.github.io/react-native/docs/appstate.html).

background - The app is running in the background. The user is either:
* in another app
* on the home screen
* [Android] on another Activity (even if it was launched by your app)

The app simply has no ability to do anything once it is closed. Think Terminating a process on your desktop. It doesn't matter what you do, once a process is terminated it can no longer do anything unless it is restarted. In the case of mobile apps, as far as I know there is no automatic way to 'restart a service' all on its own.

**Comment by Xaus** [comment]
@xerotolerant it's true but if this library is "background" timer it's supposed to use a service ...
anyway i've already made this library for my project and maybe publish my library to github sometime later .

**Comment by xerotolerant** [comment]
@Xaus Were you able to make something work cross platform? That would be cool. Re-reading the rules I suspect that triggering a timer on a service infrequently (depending on the duration of the timer) might not exceed the limit.

For example if the timer is 15mins on it, then incrementing the timer ever minute probably won't add up to more than 30 seconds of operations.

Did you do something like that in your solution?
